types/mapbox-gl/mapbox-gl-tests.ts

```diff
@@ -897,6 +897,13 @@ expectType<mapboxgl.Map>(
         expectType<undefined>(ev.originalEvent);
     }),
 );
+expectType<mapboxgl.Map>(
+    map.on('idle', ev => {
+        expectType<mapboxgl.MapboxEvent>(ev);
```

Comment:
Do we really need this expectTypes here? I'm not even sure what they are really checking. Look at ```on``` signature:

```
on(type: string, listener: (ev: any) => void): this;
```

I know, you just did it like it was done before. But that makes little sense to me, tbh. Maybe instead we should have an overload for ```on``` where type='load' ad ev=MapboxEvent